modules/gitops-release-notes-1-7-0.adoc

```diff
@@ -0,0 +1,65 @@
+| resourceActions | resource.customizations.actions.<group_kind> | resource.customizations.actions
+|===
++
+link:https://issues.redhat.com/browse/GITOPS-1561[GITOPS-1561]
++
+[NOTE]
+====
+In future releases, there is a possibility to deprecate the old method of customizing resource behavior by using only resourceCustomization and not subkeys.
+====
+
+* With this update, to use the *Environments* feature on the *Developer* tab you will be forced to upgrade if you are on {gitops-title} below 1.7 and {product-title} above or on 4.15. link:https://issues.redhat.com/browse/GITOPS-2415[GITOPS-2415] 
+
+* With this update, applications can be created in any namespace in the same cluster and still managed by the same control-plane’s ArgoCD instance. This is done by adding a new label `argocd.argoproj.io/managed-by-cluster-argocd` to the namespace added in `spec.sourceNamespaces` of the Argo CD custom resource. link:https://issues.redhat.com/browse/GITOPS-2341[GITOPS-2341]
+
+:FeatureName: Argo CD Applications controller
+include::snippets/technology-preview.adoc[]
+
+
+[id="fixed-issues-1-7-0_{context}"]
+== Fixed issues
+
+The following issues have been resolved in the current release:
+
+* Before this update, {gitops-title} releases were affected by an issue of dex pods failing with `CreateContainerConfigError` error when the `anyuid` SCC was assigned to the dex service account. This update fixes the issue by assigning a default user id to the dex container. link:https://issues.redhat.com/browse/GITOPS-2235[GITOPS-2235]
+
+* Before this update, {gitops-title} used the RHSSO (Keycloak) through OIDC in addition to Dex. However, with a recent security fix, the certificate of RHSSO could not be validated when configured with a certificate not signed by one of the well known certificate authorities. This update fixes the issue; you can provide a custom certificate to verify the KeyCloak's TLS certificate while communicating with it. In addition, you can add `rootCA` to their Argo CD custom resource ``.spec.keycloak.rootCA` field. The Operator reconciles such changes and updates the `oidc.config in argocd-cm` config map with the PEM encoded root certificate. link:https://issues.redhat.com/browse/GITOPS-2214[GITOPS-2214]
+
+* Before this update, the application controllers restarted multiple times due to the unresponsiveness of liveness probes. This update fixes the issue by removing the liveness probe in the `statefulset` application controller. link:https://issues.redhat.com/browse/GITOPS-2153[GITOPS-2153]
+
+[id="known-issues-1-7-0_{context}"]
+== Known issues
+
+* Before this update, the Operator did not reconcile the `mountsatoken` and ServiceAccount settings for the repository server. While this has been fixed, deletion of the service account does not revert to default. link:https://issues.redhat.com/browse/GITOPS-1873[GITOPS-1873]
```

Comment:
```suggestion
* Before this update, the Operator did not reconcile the `mountsatoken` and `ServiceAccount` settings for the repository server. While this has been fixed, deletion of the service account does not revert to default. link:https://issues.redhat.com/browse/GITOPS-1873[GITOPS-1873]
```

Should `ServiceAccount` be in back ticks? Or edit to say `service account`?

Reply:
back ticks